tests/scripts/run_cpp_unit_tests.sh

```diff
@@ -15,7 +15,7 @@ else
     ./build/test/tt_metal/unit_tests_fast_dispatch
     TT_METAL_GTEST_NUM_HW_CQS=2 ./build/test/tt_metal/unit_tests_fast_dispatch_single_chip_multi_queue --gtest_filter=MultiCommandQueueSingleDeviceFixture.*
     if [[ "$ARCH_NAME" == "wormhole_b0" ]]; then
-        WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml ./build/test/tt_metal/unit_tests_fast_dispatch
+        TT_METAL_GTEST_ETH_DISPATCH=1 ./build/test/tt_metal/unit_tests_fast_dispatch
```

Comment:
Can we open an issue to get rid of this too? More environment variables is something we hate seeing
@TT-billteng

Reply:
Hmm how do you want to test all fast dispatch tests with ethernet dispatch mode? Should we make a separate binary just for this toggle? I added "GTEST" to the name to make it explicit that this is a testing flag, following the scheme from `TT_METAL_GTEST_NUM_HW_CQS`.

Reply:
Shouldn't the longer cleanup be to provide this as an option for the cores through the API (which you do) and change it in the tests?

That's why I said issue to clean up later

Reply:
Overall it would be nice to see only one place with ALL constants and env names for now to keep tracking them.
for example metal_env.hpp:
`constexpr auto ETH_DISPATCH_ENV = "TT_METAL_GTEST_ETH_DISPATCH";`

Reply:
Yes we have that, it's in `rtoptions.cpp`